### PR TITLE
atomist-cli 1.1.0 (new formula)

### PR DESCRIPTION
**Pull request by ddgenome**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

This PR adds the formula for the [Atomist CLI](https://github.com/atomist/cli), which is currently provided via a dedicated [tap](https://github.com/atomist/homebrew-tap). See the [Atomist](https://atomist.com) website for more information.

### CODE REVIEW

**Comment by ddgenome** [comment]
@fxcoudert can you provide any information on how you measure "notability" and what the threshold is? This package currently gets of 1500 downloads/week from [npmjs.com/@atomist/cli](https://www.npmjs.com/package/@atomist/cli). Compare this to cash-cli.rb which is part of homebrew-core and gets 10 downloads/week according from [npmjs.com/cash-cli](https://www.npmjs.com/package/cash-cli).

**Comment by fxcoudert** [comment]
I see 6 installs per month on the current tap (via `brew info --analytics`), and the github repo has 2 watch / 3 stars / 0 fork (we usually aim for around 50 of each at a minimum).

I'm totally open to reconsidering if there is evidence of community activity / notability.

File: Formula/atomist-cli.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+    bash_completion.install "#{libexec}/lib/node_modules/@atomist/cli/assets/bash_completion/atomist"
+  end
+
+  test do
+    assert_predicate bin/"atomist", :exist?
```
Comment (fxcoudert) [inline Formula/atomist-cli.rb]: We need a test that exercises the some of the functionality of the app.

**Comment by ddgenome** [comment]
Thanks for the feedback. It's not surprising that the brew installs via the tap are low. Our [developer quick start](https://docs.atomist.com/quick-start/) instructs the user to install via `npm i -g @atomist/cli` as it is a single step versus installing via the tap.

This package provides a CLI for [Atomist SDM](https://github.com/atomist/sdm), which has 16 watch /127 stars /10 fork, and is probably a better measure of the notability of this package.

File: Formula/atomist-cli.rb (hunk 1 above)
Comment (ddgenome) [reply Formula/atomist-cli.rb]: Functionality tests added below.

**Comment by ddgenome** [comment]
Squashed, changed commit message, and rebased onto current master. Thanks!

**Comment by fxcoudert** [comment]
Thanks @ddgenome for the pull request! It's now merged

**Pull request closed without merging**